Add docs for email and sms templates for B2B orgs (#5038)
wso2/docs-is: en/includes/guides/branding/customize-sms-templates.md

The following topics explain how you can customize SMS notifications that are sent to users of your {{ product_name }} organization.

{% if product_name == "Asgardeo" %}

!!! note "SMS templates for B2B applications"

    If you have set up [organizations]({{base_path}}/guides/organization-management/manage-organizations/),
    these organizations automatically inherit the sms templates of the root organization.

{% else %}

!!! note "SMS templates for B2B applications"

    If you have set up [organizations]({{base_path}}/guides/organization-management/manage-organizations/),

>     you can customize sms templates to fit the branding needs of each organization. If you do not customize a specific

Reviewer: ```suggestion
    you can customize sms templates to fit the branding needs of each organization. If you do not customize an 
```

>     template, the sms template will be inherited from the closest ancestor organization with defined sms template.

Reviewer: ```suggestion
    sms template for an organization, it will be inherit the design from the closest ancestor organization with a customized sms template.
```

>     If no ancestor has defined sms template, the default sms template will be applied.

Reviewer: ```suggestion
    If no ancestor has customized the particular sms template, the default sms template will be applied.
```

The following topics explain how you can customize SMS notifications that are sent to users of your {{ product_name }} organization.

{% if product_name == "Asgardeo" %}

!!! note "SMS branding for B2B applications"

    If you have [organizations]({{base_path}}/guides/organization-management/manage-organizations/) configured,
    note that the sms branding you configure for your root organization also applies to your organizations.

{% else %}

> !!! note "SMS branding for B2B applications"

Reviewer 2: Is this doc completely not available in IS-7.0.0? Is that the case to not having IS version specific config

Author: Yes this is not available in IS 7.0.0
